dev/com.ibm.websphere.appserver.features/visibility/public/opentracing-1.1/resources/l10n/com.ibm.websphere.appserver.opentracing-1.1.properties

```diff
@@ -0,0 +1,16 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+################################################################################
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature provides support for Opentracing function for enterprise Java.
```

Comment:
description=This feature supports the Opentracing function for **Enterprise Java.**

Does this need to be more specific? Do we need to include the name of the feature?